practice/006/work/impl.ml

```
@@ -0,0 +1,3 @@
+let is_palindrome list =
+  list = List.rev list
```

This should throw an exception.

---

Done